server/channels/store/sqlstore/scheduled_post_store.go

```diff
@@ -0,0 +1,138 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package sqlstore
+
+import (
+	"sync"
+
+	"github.com/mattermost/mattermost/server/public/model"
+	"github.com/mattermost/mattermost/server/public/shared/mlog"
+	sq "github.com/mattermost/squirrel"
+	"github.com/pkg/errors"
+)
+
+type SqlScheduledPostStore struct {
+	*SqlStore
+	maxMessageSizeOnce   sync.Once
+	maxMessageSizeCached int
+}
+
+func newScheduledPostStore(sqlStore *SqlStore) *SqlScheduledPostStore {
+	return &SqlScheduledPostStore{
+		SqlStore:             sqlStore,
+		maxMessageSizeCached: model.PostMessageMaxRunesV2,
+	}
+}
+
+func (s *SqlScheduledPostStore) columns(prefix string) []string {
+	if prefix != "" {
+		prefix = prefix + "."
+	}
+
+	return []string{
```

Comment:
Kind of a nit, but couldn't we do this with a loop? Then we could just hardcoded the array of columns as a const. Might be a bit cleaner. 0/5

Reply:
If we use a loop, the const array of columns won't be useful as we'd always be creating a new one in the loop. Anyways, I'm just trying this approach of function with prefix, lets see how it goes.